docs/api.md

```diff
@@ -30,7 +30,7 @@ curl -X POST localhost:8081/frontends/myhost.com%2Ftest/mybackend
 
```

Comment:
if trailing slashes are now required, this POST/PUT request (line 28) is now broken, thus the api is broken